### PR TITLE
Feature/j 01 klantbeeld gekoppelde zaken overzicht

### PR DESCRIPTION
**Pull request by lencodes**
**Note: merging this into `feature/Q-01_...`.**

This pull request includes:

- Addition and implementation of `ZakenOverzichtKlantbeeld` component;
- Refactoring of `findByBsn` method by wrapping it in `fetcher`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,4 +50,4 @@
     "vitest": "^0.9.3",
     "vue-tsc": "^0.34.7"
   }
-}
+}
```
Comment (lencodes) [inline package.json:53]: This was done by the automatic formatter.

File: src/features/zaaksysteem/ZakenOverzichtKlantbeeld.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const zaakService = useZaaksysteemService();
+
+const zaken = zaakService.findByBsn(klantBsn).withFetcher();
+</script>
+
+<style lang="scss" scoped>
+.container {
+  margin-inline: var(--spacing-extralarge);
+}
```
Comment (lencodes) [inline src/features/zaaksysteem/ZakenOverzichtKlantbeeld.vue:60]: Some odd layouts are already happening, I think we should be re-thinking the way we're setting up our containers. Right now, it seems like every component (or template) creates its own margins to mimic a container.

Might be better to have an actual `<container>` element which renders its children within itself.

File: src/services/formatDate.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { DateTime } from "luxon";
+
+export const formatDate = (date: string): string => {
```
Comment (felixcicatt) [inline src/services/formatDate.ts:3]: We don't use this same format everywhere. You (rightly) brought up that that's something we might want to do. Until then, can you to specify this function name a bit more?

File: src/views/ContactmomentView.vue [hunk 4]
```diff
@@ -246,4 +252,12 @@ aside {
     }
   }
 }
+
+.informatie-container {
```
Comment (felixcicatt) [inline src/views/ContactmomentView.vue:256]: I think we might not need this div. If you put display:grid on .klant-panel, with a gap of var(--spacing-large), you get the same effect. This makes the 'Klanten zoeken' button drop a bit, but I think that doesn't look bad.

Comment (felixcicatt) [reply src/views/ContactmomentView.vue:256]: Also, unrelated to your PR: if you change `ul li article` to `ul li > article` on line 196, this fixes the margins in the customer details panel.

**Review by felixcicatt — CHANGES_REQUESTED** [review]

File: src/features/zaaksysteem/ZakenOverzichtKlantbeeld.vue (hunk 2 above)
Comment (felixcicatt) [reply src/features/zaaksysteem/ZakenOverzichtKlantbeeld.vue:60]: Agreed. Let's wait until the design for the layout has matured a bit, and then discuss a better layout system in code.

**Review by felixcicatt — APPROVED** [review]

**Pull request merged**